crawl-ref/source/tilepick.cc

```diff
@@ -1933,6 +1933,19 @@ tileidx_t tileidx_monster(const monster_info& mons)
         ch |= TILE_FLAG_GD_NEUTRAL;
     else if (mons.neutral())
         ch |= TILE_FLAG_NEUTRAL;
+    else if (mons.threat == MTHRT_TRIVIAL &&
```

Comment:
crawl coding style is to use braces for each statement here, because the test takes more than one line:

```c
    else if (mons.threat == MTHRT_EASY &&
        Options.tile_show_threat_levels.find("easy") != string::npos)
    {
            ch |= TILE_FLAG_EASY;
    }
```

Adds a lot of extra lines here though...

Reply:
https://github.com/crawl/crawl/blob/master/crawl-ref/docs/develop/coding_conventions.txt

Reply:
Hmm. I hate switch() and never use it in code of my own, but given crawl's conventions, maybe it would be better, as to avoid the extra lines you refer to? Like so:

```
else
{
    switch (mons.threat)
    {
    case MTHRT_EASY:
        if (Options.tile_show_threat_levels.find("easy") != string::npos)
            ch |= TILE_FLAG_EASY;
        break;
    [etc.]
    }
}
```

Let me know if you think that would do, and if so I'll check it in.

Reply:
Checked in the switch-case version a while back. Maybe I'm supposed to hit "Resolve conversation." Not sure whether it matters that much, but going to press it just in case. :)